E2E-tests/src/sidechain_main_cli.py

```diff
@@ -81,6 +81,42 @@ def get_signatures(
             raise e
         return signatures
 
+    def update_d_param(
+        self,
+        permissioned_candidates_count,
+        trustless_candidates_count,
+        payment_key,
+    ):
+        update_d_param_cmd = (
+            f"{self.cli} update-d-parameter "
+            f"--genesis-committee-hash-utxo {self.config.genesis_committee_hash_utxo} "
+            f"--governance-authority {self.config.governance_authority} "
+            f"--threshold-numerator {self.config.threshold_numerator} "
+            f"--threshold-denominator {self.config.threshold_denominator} "
+            f"--sidechain-id {self.config.chain_id} "
+            f"--atms-kind {self.config.atms_kind} "
```

Comment:
how did you get it working?
`src.sidechain_main_cli.SidechainMainCliException: Invalid option '--atms-kind'`

Reply:
Hey @rsporny , I didn't realize that this test has the `probability` marker, which we skip. Is that right? Should I remove that marker?

Reply:
yes, please remove it